JitsiMediaDevices.js

```diff
@@ -138,6 +138,18 @@ const JitsiMediaDevices = {
      */
     emitEvent(event, ...args) {
         eventEmitter.emit(event, ...args);
+    },
+
+    /**
+     * Returns whether or not the current browser can support capturing video,
+     * be it camera or desktop, and displaying received video.
+     *
+     * @returns {boolean}
+     */
+    supportsVideo() {
+        // Defer to RTCBrowserType to allow exposure of the api to the consumer
+        // but prevent other files from having to import JitsiMediaDevices.
+        return RTCBrowserType.supportsVideo();
```

Comment:
can we just expose RTCBrowserType?

Reply:
I remember we had a discussion at some point about exporting RTCBrowserType so we can use it in jitsi-meet. We decided not to do that ... but I don't remember details.